### ISSUE TITLE
Headless

### ISSUE THREAD

**Issue by Massimiliano-solutiontech** [tauri-apps/wry#963]
I would like to use WRY in order to make some automated tasks currently I’m using the `Node.js` library [Puppeteer](https://github.com/puppeteer/puppeteer) in headless mode but I would like to switch to `Rust` so I wonder if I can use `WRY` + [`fantoccini`](https://github.com/jonhoo/fantoccini)

**Describe alternatives you've considered**
[This](https://github.com/rust-headless-chrome/rust-headless-chrome) project is an alternative

**Would you want to assign yourself to implement this feature?**
- [ ] Yes
- [x] No

**Comment by FabianLars** [comment]
This was already mentioned/discussed a few times here: https://github.com/tauri-apps/wry/discussions?discussions_q=is%3Aopen+headless and also kinda here: https://github.com/tauri-apps/wry/issues/677.

tldr (i'd still suggest looking through all the comments): Kindaaa out of scope of wry but you _can_ use it to an extend.

that said, i'm not sure how webdriver compatible it is, or how tauri-specific [`tauri-driver`](https://github.com/tauri-apps/tauri/tree/dev/tooling/webdriver) is.

either way, we can't spend any resources on this use-case, sorry. (but feel free to report back any findings and issues you have!)
